references/optical_flow/transforms.py

```diff
@@ -0,0 +1,267 @@
+import torch
+import torchvision.transforms as T
+import torchvision.transforms.functional as F
+
+
+class ValidateModelInput(torch.nn.Module):
```

Comment:
For some of these (e.g. `valid_flow_mask`) we wouldn't be able to make the assertion in the model, because the model doesn't use it as input - it's only used for the loss computation.

Similarly we assert that the dtype is float32 but we wouldn't want to leave this in the model. I know this is somewhat exotic w.r.t. the rest of the references, but would it be OK to leave it (possibly commenting it out from the Presets? I feel like this adds value at least as documentation

Reply:
Yeah that's fine. It's not a big issue.